libdeno/api.cc

```diff
@@ -108,10 +108,11 @@ deno_buf deno_get_snapshot(Deno* d_) {
 static std::unique_ptr<v8::Platform> platform;
 
 void deno_init() {
-  CHECK_NULL(platform.get());
-  platform = v8::platform::NewDefaultPlatform();
-  v8::V8::InitializePlatform(platform.get());
-  v8::V8::Initialize();
+  if (platform.get() == nullptr) {
```

Comment:
I'm not sure if this will cause problems. It seems to work fine this way, but most of the time there is a reason for things like this.

Reply:
I'm not sure either - but if the tests pass (which they do) then I'm ok with it.